src/api/public/apidocs-new/paths/source_project_name_pubkey.yaml

```
@@ -0,0 +1,23 @@
+get:
+  summary: Get GPG key
+  description: |
+    Get the project's GPG key. If the project doesn't have a default GPG key, it looks for the first one
+    available in the namespace hierarchy, ending at the global buildservice key.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: gpgkey
```

```suggestion
            example: |
              -----BEGIN PGP PUBLIC KEY BLOCK-----
              Version: GnuPG v1.4.5 (GNU/Linux)

              mQENBFfX+W4BCAC89VZEUt8O/o2rXnFoNrW0nsZoaHZX5Kjtzwuns5/UwzBffY/h
              [...]
              -----END PGP PUBLIC KEY BLOCK-----
```

Even not being a full PGP public key, I would change the example of the text to something more similar.